### PR TITLE
Hotfix: Incorrect timers computation

### PR DESCRIPTION
**Pull request by quintanilhar**
Release notes:

**Backport:**
- Fix [TR-1857](https://oat-sa.atlassian.net/browse/TR-1857) Every next attempt duration for the first item is the summary of the time spent per attempt plus all time spent in test [#1947](https://github.com/oat-sa/extension-tao-testqti/pull/1947), [#1949](https://github.com/oat-sa/extension-tao-testqti/pull/1949)

### CODE REVIEW

File: views/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,29 @@
 {
   "name": "@oat-sa/tao-testqti",
   "version": "0.3.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (alexeyvy) [inline views/package-lock.json:4]: I don't see this change in the PRs we are back-porting

Comment (alexeyvy) [reply views/package-lock.json:4]: it may be safer not to touch it :)

**Review by alexeyvy — APPROVED** [review]

File: views/package-lock.json (hunk 1 above)
Comment (quintanilhar) [reply views/package-lock.json]: Agreed, reverted.

File: models/classes/runner/session/TestSession.php [hunk 2]
```diff
@@ -204,7 +204,7 @@ public function initItemTimer($timestamp = null)
         // try to close existing time range if any, in order to be sure the test will start or restart a new range.
         // if the range is already closed, a message will be added to the log
         $tags = $this->getItemTags($this->getCurrentRouteItem());
-        $this->getTimer()->start($tags, $timestamp)->save();
+        $this->getTimer()->end($tags, $timestamp)->save();
```
Comment (m-lagarde) [inline models/classes/runner/session/TestSession.php:207]: don't we have tests for that? :thinking:

Comment (quintanilhar) [reply models/classes/runner/session/TestSession.php:207]: For the timer, yes but not for this class. I can't find any test related to this method.

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**